dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -63,7 +63,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
     }
 
-    winrt::IAsyncOperationWithProgress<winrt::Microsoft::Windows::PushNotifications::PushNotificationCreateChannelResult, winrt::Microsoft::Windows::PushNotifications::PushNotificationCreateChannelStatus> PushNotificationManager::CreateChannelAsync(const winrt::guid &remoteId)
+    winrt::IAsyncOperationWithProgress<winrt::Microsoft::Windows::PushNotifications::PushNotificationCreateChannelResult, winrt::Microsoft::Windows::PushNotifications::PushNotificationCreateChannelStatus> PushNotificationManager::CreateChannelAsync(const winrt::guid& remoteId)
     {
```

Comment:
I saw another PR like this - any member method that uses co_await needs to capture a strong reference to "this" like:

```c++
IAsyncOperation<Q> Foo::ThingAsync()
{
    auto lifetime = get_strong();
    // the rest of the code
}
```

... without this you can accidentally final-deref the type while an async operation is running.